CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # MicroProfile Language Server Changelog
 
+## [0.12.0](https://github.com/eclipse/lsp4mp/milestone/13?closed=1) (August 14, 2024)
+
+### Enhancements
+
+- Add support for @Startup healthcheck diagnostic. See [#443](https://github.com/eclipse/lsp4mp/issues/443).
+
+### Bug Fixes
+
+- Unrecognized property mp.messaging.* when Channel annotation is used along with Multi. See [#451](https://github.com/eclipse/lsp4mp/pull/451).
+- fix: synchronized projectLabels + collect codeLens when project is loaded. See [#444](https://github.com/eclipse/lsp4mp/pull/444).
+
+### Build
+
+- Update to the latest jdt-ls. See [#446](https://github.com/eclipse/lsp4mp/issues/446).
```

Comment:
Note here that we are upgrading to 1.37.0-SNAPSHOT

Reply:
Good catch!

It seems the upgrade is 1.39.0-SNAPSHOT.